### DISCUSSION TITLE
Two-Factor authentication - enforcement

### DISCUSSION THREAD

**Discussion by boris-ning-usds** [CDCgov/template#29] [Announcements]
Good afternoon,

The maintainers of github.com/cdcgov will be turning on 2FA authentication enforcement for all accounts logging into github.com/cdcgov on May 1st 2023. Accounts logging into cdcgov won't be able to access github.com/cdcgov does not have 2FA associated with it.

Feel free to read [how to setup 2FA with your Github account](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa) on how to set it up.

tl;dr - Go to https://github.com/settings/security and turn it on and save/store the QR code or secret token somewhere, and be sure to save a backup.
![image](https://user-images.githubusercontent.com/41026275/232861788-df870d54-cf58-4c44-b0cc-953704cf9312.png)
For additional support, don't hesitate to file a [Github issue](https://github.com/CDCgov/template/issues/new/choose) for help.

-Boris
